src/stream/src/executor/hash_agg.rs

```diff
@@ -84,19 +83,22 @@ struct HashAggExecutorExtra<S: StateStore> {
     /// A [`HashAggExecutor`] may have multiple [`AggCall`]s.
     agg_calls: Vec<AggCall>,
 
+    /// Relational state tables for each aggregation calls.
+    /// `None` means the agg call need not to maintain a state table by itself.
+    agg_state_tables: Vec<Option<AggStateTable<S>>>,
```

Comment:
maybe in the future, we need to change it to enum type when we add some agg call's states are neither its result nor a table with some materialized columns of AggOerator's input.

Reply:
+1. Defining an enum for this gives clearer semantics.